app/controllers/api/health.php

```diff
@@ -253,41 +319,51 @@
             }
         }
 
-        $response->json(['status' => 'OK']);
+        $output = [
+            'status' => 'pass',
+            'checkTime' => \microtime(true) - $checkStart
+        ];
+
+        $response->dynamic(new Document($output), Response::MODEL_HEALTH_STATUS);
     });
 
 App::get('/v1/health/anti-virus')
-    ->desc('Get Anti virus')
+    ->desc('Get AntiVirus')
     ->groups(['api', 'health'])
     ->label('scope', 'health.read')
     ->label('sdk.auth', [APP_AUTH_TYPE_KEY])
     ->label('sdk.namespace', 'health')
     ->label('sdk.method', 'getAntiVirus')
     ->label('sdk.description', '/docs/references/health/get-storage-anti-virus.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_OK)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_HEALTH_ANTIVIRUS)
     ->inject('response')
     ->action(function ($response) {
         /** @var Appwrite\Utopia\Response $response */
 
+        $output = [
+            'status' => '',
+            'version' => ''
+        ];
+
         if (App::getEnv('_APP_STORAGE_ANTIVIRUS') === 'disabled') { // Check if scans are enabled
-            return $response->json([
-                'status' => 'disabled',
-                'version' => '',
-            ]);
+            $output['status'] = 'disabled';
+            $output['version'] = '';
+        } else {
+            $antiVirus = new Network(App::getEnv('_APP_STORAGE_ANTIVIRUS_HOST', 'clamav'),
+                (int) App::getEnv('_APP_STORAGE_ANTIVIRUS_PORT', 3310));
+
+            try {
+                $output['version'] = @$antiVirus->version();
```

Comment:
What is @antivirus  👀

Reply:
No idea 👀 Just copied what was already there

Reply:
Okay I don't know what this is either 😆 I thought you knew .

Reply:
Its used to silent errors. Its used all over the code, let's just stick with it.